etna/transforms/math/differencing.py

```diff
@@ -73,6 +76,14 @@ def _get_column_name(self) -> str:
         else:
             return self.out_column
 
+    def _validate_segments(self, segments: List[str]):
```

Comment:
Looks like come general function which might be helpfull for several transforms, maybe we can put it in utils?